runtime/ftplugin/query.lua

```diff
@@ -2,5 +2,21 @@
 -- Language:	Tree-sitter query
 -- Last Change:	2022 Mar 29
 
+if vim.b.did_ftplugin == 1 then
+  return
+end
+
+if vim.g.query_disable_linter ~= 1 then
```

Comment:
the query linter should be disabled for `vim.treesitter.inspect_tree`. Since it doesn't know the lang of this buffer it will anyways just do syntax linting.

Reply:
Or hardcode `query` language for `query` filetype?

Reply:
#22887 suggests alternative filetype for playground. Disabled it manually until #22887 ismerged